spce/spce.go

```diff
@@ -0,0 +1,118 @@
+package spce
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+)
+
+type Code uint8
+
+const (
+	Current  Code = 0x0a
+	Pressure Code = 0x0b
+	Voltage  Code = 0x0c
+)
+
+type Status string
+
+const (
+	Ok Status = "OK"
+	Er Status = "ER"
+)
+
+type SPCE struct {
+	rw      io.ReadWriter
+	channel uint8
+}
+
+// NewSPCE returns a new SPCE instance on a given ReadWriter for the specified channel.
+func NewSPCE(rw io.ReadWriter, channel uint8) *SPCE {
```

Comment:
I changed the type of `channel` to `uint8` to reduce the space for errors.

Reply:
I also think we should not use default parameters here, in the sense of don't assume something you don't know. Better someone thinks a minute or two what the `channel` is on their device then wondering why it doesn't work because someone many years ago changed the value.